tasks/db_tasks.py

```diff
@@ -98,7 +98,7 @@ def seed_dev_db(c):
                 f" {apps * rounds * funds} test rows."
             )
 
-            seed_database(apps, rounds, funds)
+            seed_database_randomly(apps, rounds, funds)
```

Comment:
Why are we making this change in this PR? Seems unrelated?

Reply:
Think it was re-named but not committed in another PR, I added it so I could run the script and committed it at the same time for anyone else running it (incase they hit the same issue)